subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -130,6 +132,17 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	if (num_bis != 1) {
 		return BT_HCI_ERR_MEM_CAPACITY_EXCEEDED;
 	}
+
+	/* Add ACAD to AUX_SYNC_IND */
+	err = ull_adv_sync_acad_enable(lll_adv_sync,
+				       (sizeof(struct pdu_big_info) + 2),
```

Comment:
I'm assuming that the `+ 2` is for the length field and AD type. Don't we have any AD header struct that we can call the `sizeof()` on, or a macro that better describes this value we can use instead?

Reply:
Isnt this old code?

Reply:
It is. I was reviewing commit-by-commit. You can disregard this.